server/config/routes/api/v3.rb

```diff
@@ -0,0 +1,36 @@
+Fusor::Engine.routes.draw do
```

Comment:
@cfchase, the `foreman_jsonapi` gem is using v21, since the foreman community didn't want to call it v3. I agree that it's changed significantly, but a lot of other stuff has changed from QCI 1.0 to 1.1, so I don't see the advantage of renaming to v3. If a person installs 1.0, it will use `foretello_api_v21` and if 1.1, it will use `foreman_jsonapi`